_posts/2017-09-01-tips-for-upgrading-rails-3-2-to-4.markdown

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title: "The definitive guide to upgrading from Rails 3.2 to 4.0"
+date: 2017-09-01 16:48:00
+categories: ["rails", "upgrades"]
+author: "mauro-oto"
+---
+
+_This article is part of our Upgrade Rails series. To see more of them, [click here](https://www.ombulabs.com/blog/tags/upgrades)_.
+
+A [previous post](https://www.ombulabs.com/blog/rails/tips-for-upgrading-rails-3-2-to-4.html)
+covered some general tips to take into account for this migration. This article
+will try to go a bit more in depth. We will first go from 3.2 to 4.0, and then
+to 4.2. Depending on the complexity of your app, a Rails upgrade can take
+anywhere from one week for a single developer, to a few months for two developers.
+
+1. Ruby version
+2. Gems
+3. Config files (config/)
+4. Application code
+  a. Models (app/models/)
```

Comment:
the markdown preview is not showing this in a new line

Reply:
Fixed, thanks